File: jan14/jan14.java
import java.util.Scanner;

public class MultiplicationTable {
    public static void generateMultiplicationTable(int number, int rangeLimit) {
        System.out.printf("Multiplication table for %d up to %d * %d:\n\n", number, number, rangeLimit);
        for (int i = 1; i <= rangeLimit; i++) {
            System.out.printf("%d * %d = %d\n", number, i, number * i);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = scanner.nextInt();
        System.out.print("Enter the range: ");
        int rangeLimit = scanner.nextInt();

        generateMultiplicationTable(number, rangeLimit);
        scanner.close();
    }
}

File: jan14/jan14.js
function generateMultiplicationTable(number, rangeLimit) {
    console.log(`Multiplication table for ${number} up to ${number} * ${rangeLimit}:\n`);
    for (let i = 1; i <= rangeLimit; i++) {
        console.log(`${number} * ${i} = ${number * i}`);
    }
}

function main() {
    const readline = require('readline');
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    rl.question('Enter a number: ', (number) => {
        rl.question('Enter the range: ', (rangeLimit) => {
            generateMultiplicationTable(parseInt(number), parseInt(rangeLimit));
            rl.close();
        });
    });
}

main();